External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -1395,7 +1363,7 @@ private static void EventMetatag(ClassModel inClass, ScintillaControl Sci, Membe
             InsertCode(position, template);
         }
 
-        private static void ConvertToConst(ClassModel inClass, ScintillaControl Sci, MemberModel member, bool detach)
+        private static void ConvertToConst(ClassModel inClass, ScintillaControl Sci, MemberModel member, bool detach, int style)
```

Comment:
You could probably re-evaluate the style at cursor position instead of doing all these changes to pass `style`.